packages/open-next/src/adapters/routing/middleware.ts

```diff
@@ -32,7 +32,7 @@ type MiddlewareOutputEvent = InternalEvent & {
 
 //    if res.end() is return, the parent needs to return and not process next server
 export async function handleMiddleware(
-  internalEvent: InternalEvent,
+  internalEvent: InternalEvent
```

Comment:
There is a bunch of changes like that inside your PR.
It might be your local prettier messing with our eslint config. You can run `pnpm run lint:fix` to solve this